cmd/mdatagen/main_test.go

```diff
@@ -29,7 +29,7 @@ name: metricreceiver
 metrics:
   system.cpu.time:
     description: Total CPU seconds broken down by different states.
-    extended_description: Additional information on CPU Time can be found [here](https://en.wikipedia.org/wiki/CPU_time).
+    extended_documentation: Additional information on CPU Time can be found [here](https://en.wikipedia.org/wiki/CPU_time).
```

Comment:
This change is required to pass new mapstructure validation. Submitted another PR https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/6473